README.md

```diff
@@ -8,21 +8,21 @@ Software developer tools that facilitate the interaction with the Safe [contract
 
 ## Guides
 
-| Title | Description |
+| Title                                                                                                                           | Description                                                                                                                                                                                                          |
 | ------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Same ⬆️

Reply:
This is an autoformatter, like in line 12. I don't like how it does it either, would be good if we could avoid it in the tables inside all the repo

Reply:
removed!